templates/product/details.html

```diff
@@ -159,7 +159,7 @@ <h2 class="product__info__price">
       <div class="product__info__description">
         <h3>{% trans "Description" context "Product details title" %}</h3>
         <hr>
-        {{ product.description|markdown }}
+        {{ product.description|safe }}
```

Comment:
This would need a migration to convert existing markdown to HTML.

Reply:
Summon @elwoodxblues